components/TransactionList/TransactionList.styled.js

```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+
+export const Ul = styled.ul`
+  display: flex;
+  flex-direction: column;
+  width: min(100% - 30px, 395px - 30px);
```

Comment:
Just something to think about: 
Hard coded values like this are often called "Magic numbers". One problem with magic numbers is they have a lack of context and they are not self explaining. Also they can be hard to update if they appear at multiple places in your code and your requirements change. 
One solution is to set variables for them, like this they provide context and are easy to update. 

```
const doubledBorder = "30px";
const minimalDeviceWidth = "395px";
```
```
  width: min(100% - doubledBorder, minimalDeviceWidth - doubledBorder);
```